tests/Header.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import Header from '../src/ui/Header';
+import { Navigator, ReaderState } from '../src/types';
+import { Icon, IconNames } from '@nypl/design-system-react-components';
+
+test('render header bar', () => {
+  const readerState = {} as ReaderState;
+  const navigator = {} as Navigator;
+
+  render(<Header readerState={readerState} navigator={navigator} />);
+
+  expect(
+    screen.getByRole('link', { name: 'Return to NYPL' })
+  ).toBeInTheDocument();
+
+  expect(screen.getByRole('link', { name: 'Return to NYPL' })).toHaveAttribute(
+    'href',
+    'https://www.nypl.org'
+  );
+
+  expect(screen.getByRole('button', { name: 'Settings' })).toBeInTheDocument();
+
+  expect(
+    screen.getByRole('button', { name: 'Toggle Fullscreen' })
+  ).toBeInTheDocument();
+});
+
+test('render custom left header', () => {
+  const readerState = {} as ReaderState;
+  const navigator = {} as Navigator;
+  const IconComponent = () => (
+    <span data-testid="custom-icon">
```

Comment:
It's totally find to use testId here because it's scoped just to the test.  It's only really a smell when you're finding yourself putting testIds on stuff outside of the test, such as the "toggle fullscreen" button